cmd/helm/repo_list.go

```diff
@@ -38,7 +38,7 @@ func newRepoListCmd(out io.Writer) *cobra.Command {
 		Args:    require.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			f, err := repo.LoadFile(settings.RepositoryConfig)
-			if isNotExist(err) || len(f.Repositories) == 0 {
```

Comment:
I think this should use `output.JSON` and `output.YAML` instead of the string literals. Otherwise, looks great!

Reply:
@technosophos , Yes, I have made the changes.